### PR TITLE
Add title and aria-hidden

### PR DESCRIPTION
**Pull request by thedaviddias**
I added `aria-hidden` and a `title` attribute to be more accessible.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -28,12 +28,13 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
 var Icon = function Icon(_ref) {
   var icon = _ref.icon,
       flex = _ref.flex;
-  return _react2.default.createElement('span', { className: (0, _classnames3.default)(_styles2.default.icon, _styles2.default[icon], _defineProperty({}, '' + _styles2.default.flex, flex)) });
+  return _react2.default.createElement('i', { className: (0, _classnames3.default)(_styles2.default.icon, _styles2.default[icon], _defineProperty({}, '' + _styles2.default.flex, flex)) });
```
Comment (SeeThruHead) [inline lib/index.js]: why is this an i tag when you haven't changed the source code to be an i tag?

**Review by SeeThruHead — COMMENTED** [review]
what does that have to do with my questions :D

File: src/index.js [hunk 2]
```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types';
...
-  <span className={
-    classnames(styles.icon, styles[icon], {[`${styles.flex}`]: flex})
-  } />;
-
+const Icon = ({icon, ariaHidden, title, flex}) =>
+  <span
+    className={
+      classnames(styles.icon, styles[icon], {[`${styles.flex}`]: flex})
+    }
+    aria-hidden
```
Comment (SeeThruHead) [inline src/index.js:11]: props in react are camelCased

File: lib/index.js (hunk 1 above)
Comment (thedaviddias) [reply lib/index.js]: The build didn't change it... I'm launching it again.

File: src/index.js (hunk 2 above)
Comment (thedaviddias) [reply src/index.js:11]: not aria: https://reactjs.org/docs/dom-elements.html

**Review by SeeThruHead — APPROVED** [review]

**Pull request merged**